backend/src/main/java/com/capstone/moneytree/model/SanitizedUser.java

```diff
@@ -0,0 +1,86 @@
+package com.capstone.moneytree.model;
+
+import com.capstone.moneytree.model.node.User;
+
+public class SanitizedUser { // this class will be used to return only the necessary info about a user for follower/followings api and other apis
```

Comment:
To avoid duplicates, would this be better if we created a new constructor in the User model class with all the necessary parameters instead of creating a new class? And then you would get a SanitizedUser with a new getter as well only getting the needed info for the user.

Reply:
Very good idea, I agree, that is the right way of doing that but didn't come to my mind, I will change it in my next commit.

Reply:
Very good idea, I agree, that is the right way of doing that but didn't come to my mind, I will change it in my next commit.

Reply:
Very good idea, I agree, that is the right way of doing that but didn't come to my mind, I will change it in my next commit.